components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -128,27 +146,58 @@ export default function () {
                             })}
                             {activeWorkspaces.length > 0 && <div className="py-6"></div>}
                             {inactiveWorkspaces.length > 0 && (
-                                <div className="p-3 text-gray-400 bg-gray-50 dark:bg-gray-800 rounded-xl text-sm text-center">
-                                    Unpinned workspaces that have been inactive for more than 14 days will be
-                                    automatically deleted.{" "}
-                                    <a
-                                        className="gp-link"
-                                        href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection"
+                                <div className="pt-14">
+                                    <div className="border-t border-gray-200 dark:border-gray-800"></div>
+                                    <div
+                                        onClick={() => setShowInactive(!showInactive)}
+                                        className="flex cursor-pointer py-6"
                                     >
-                                        Learn more
-                                    </a>
+                                        <div className="flex flex-col">
+                                            <h2 className="">Inactive Workspaces</h2>
```

Comment:
**suggestion**: Regarding the question from https://github.com/gitpod-io/gitpod/pull/10450#issuecomment-1145817205, we can keep the existing behavior, that is, removing the empty state when there are no active workspaces.

| User with only inactive workspaces |
|-|
| <img width="1440" alt="InactiveWorkspaces" src="https://user-images.githubusercontent.com/120486/172184227-4bb5ef67-61e6-46cf-b4ad-9eb359301e73.png"> |

Reply:
Marking as resolved. ✔️